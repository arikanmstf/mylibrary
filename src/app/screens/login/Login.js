/**
 * Screen Template By => create-module script
 * @version 1.0.0
 *
 */

// @flow
import * as React from 'react';
import t from 'helpers/i18n/Translate';
import type { LoginProps } from './LoginTypes';

// eslint-disable-next-line
class Login extends React.Component<LoginProps> {
  render() {
    const {
      Text, View,
    } = this.props;

    return (
      <View>
        <Text>{t.get('welcome')}</Text>
      </View>
    );
  }
}

export default Login;
