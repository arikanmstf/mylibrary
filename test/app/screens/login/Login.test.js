/**
 * Screen Test Template By => create-module script
 * @version 1.0.0
 *
 */

import React from 'react';
import { shallow } from 'enzyme';
import Login from 'screens/login/Login';

describe('test/app/screens/login/Login.test.js', () => {
  const props = {};

  it('Render', () => {
    const wrapper = shallow(<Login {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
