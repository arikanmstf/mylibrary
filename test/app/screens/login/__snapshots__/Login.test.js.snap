// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test/app/screens/login/Login.test.js Render 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Login
    initialize={[MockFunction]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <UNDEFINED>
        welcome
      </UNDEFINED>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": "welcome",
      },
      "ref": null,
      "rendered": "welcome",
      "type": undefined,
    },
    "type": undefined,
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <UNDEFINED>
          welcome
        </UNDEFINED>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": "welcome",
        },
        "ref": null,
        "rendered": "welcome",
        "type": undefined,
      },
      "type": undefined,
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
